.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
 version: 2
 
 jobs:
-  build:
-    working_directory: /home/reaction
+  # run automated tests
+  tests:
+    working_directory: ~/reaction
 
     docker:
-      - image: node:8
+      - image: circleci/node:8.9.3-browsers
```

Comment:
Changing this per comment made [here](https://github.com/reactioncommerce/reaction/issues/2858#issuecomment-360261622)